ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -379,6 +379,7 @@ literals:
   - OCCUPIED
   - UNKNOWN
   - UNOCCUPIED
+  - INACTIVE
```

Comment:
Whats the difference between `UNOCCUPIED` and `INACTIVE`?

Reply:
UNKNOWN: Unknown is state which is caused due to some kind of fault/error example: battery fault

INACTIVE: It is the state which indicates that the sensor output is not taken into consideration. example for a particular area and for particular time schedule the sensor output is not taken into consideration and the lights are always turned ON or OFF.

Reply:
these seem mutually exclusive. one applies to the state of the zone (occupied or unoccupied) and one applies to the light itself (run_status). i think these should be separated and INACTIVE not added as a state to the zone_occupancy_status.

Reply:
This "INACTIVE" state is for the zone itself which means that the irrespective of the output as a state which can be "OCCUPIED" and "UNOCCUPIED" is not taken into consideration for that particular zone.

Reply:
but shouldnt these be separate fields? i expect the `zone_occupancy_status` to tell me whether the zone is occupied or unoccupied, and thats it. you cannot override a zone status. if this were a `zone_command` (or something similar, i dont believe this is defined yet), then your states might make more sense... the zone is commanded to occupy, unoccupy, or inactive. i would still argue this is not a correct representation because the zone can either be commanded active or inactive -- occupied is active and inactive/unoccupy represent the same thing (inactive).

is it possible to separate this functionality into two fields?

Reply:
Hi Trevor,

Something is creating confusion. To make you understand the functionality i have attached the snapshot of the state description provided by the vendor I believe this will help. For more detailed understanding attaching the document too.

[https://www.lutron.com/TechnicalDocumentLibrary/369998.pdf]
(![image](https://user-images.githubusercontent.com/104188629/187075577-4b414b0d-e151-4008-8e34-dad886911164.png))

Reply:
Done the required changes

Reply:
yes but you added it below on line 399. good